google-beta/resource_compute_network.go

```diff
@@ -24,46 +48,39 @@ func resourceComputeNetwork() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
-
 			"auto_create_subnetworks": {
 				Type:     schema.TypeBool,
 				Optional: true,
 				ForceNew: true,
 				Default:  true,
 			},
-
 			"description": {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
 			},
-
-			"routing_mode": {
+			"ipv4_range": {
 				Type:     schema.TypeString,
 				Optional: true,
-				Computed: true,
+				ForceNew: true,
```

Comment:
Needs a deprecated value.

Reply:
Filed https://github.com/GoogleCloudPlatform/magic-modules/issues/1490 against MM; I'd rather do that in a separate PR. I'd like to do it as a followup to logically separate the [MM] features in history. What do you think?

Reply:
That's totally all right, but it seems like a pretty easy change, and there's something aesthetically unpleasant about a feature being un-deprecated before it's deprecated again.  Not a sticking point for me.

Reply:
Eh, fair enough. I'll add it to the existing PR.